@pauliescanlon/gatsby-mdx-embed/src/components/GeneralObserver/GeneralObserver.tsx

```diff
@@ -9,26 +9,28 @@ import React, {
 interface GeneralObserverProps {
   /** Fires when IntersectionObserver enters viewport */
   onEnter?: (id?: string) => void
+  height?: number
```

Comment:
Would you mind adding a small description for `height`, I know it's obvious but just for completeness.

Reply:
Great call, can do!